Libraries/Network/NetInfo.js

```diff
@@ -168,6 +168,9 @@ const _isConnectedSubscriptions = new Map();
  * ```
  */
 const NetInfo = {
+  /*
+   * Invokes the handler whenever the network info changes. The handler is passed with any of the connectivity types listed above.
```

Comment:
This is for markdown files in the docs section. We  wrap doc blocks at ~80 for js files.